client/CS130_Team13/Assets/Scripts/RemoteController.cs

```diff
@@ -7,75 +7,129 @@
 // serializable means it can be converted to JSON info easily 
 // https://docs.unity3d.com/Manual/JSONSerialization.html
 [System.Serializable]
-public class UserInfo
-{
-  public string name;
-  public int id;
-  public UserInfo(string name, int id)
-  {
-    this.name = name;
-    this.id = id;
-  }
-  public override string ToString()
-  {
-    return "name: " + name + ", id: " + id;
-  }
+public class UserInfo {
+    public string name;
+    public int id;
+
+    public UserInfo(string name, int id) {
+        this.name = name;
+        this.id = id;
+    }
+
+    public override string ToString() {
+        return "name: " + name + ", id: " + id;
+    }
 }
 
-public class RemoteController : MonoBehaviour
-{
-  private QSocket socket;
-  private UserInfo userInfo;
-  private UserInfo opponentInfo;
-
-  void Start()
-  {
-    StartCoroutine(InitializeGame());
-  }
-
-  IEnumerator InitializeGame()
-  {
-    int id;
-    using (UnityWebRequest webRequest = UnityWebRequest.Get("http://localhost:3000/user_id"))
-    { // TODO: ERROR CHECK
-      // Request and wait for the desired page.
-      yield return webRequest.SendWebRequest();
-      id = System.Convert.ToInt32(webRequest.downloadHandler.text);
-      Debug.Log(webRequest.downloadHandler.text);
+public class RemoteController : MonoBehaviour {
+    private QSocket socket;
+    private UserInfo userInfo;
+    private UserInfo opponentInfo;
+
```

Comment:
constructor in future (created by GameManager)
```suggestion
private string username;
private final const SERVER_URL = "http://localhost:3000";
/**
* public RemoteController(string username) {}
**/
```

Reply:
added